src/panels/lovelace/editor/card-editor/hui-card-preview.ts

```diff
@@ -35,6 +36,10 @@ export class HuiCardPreview extends HTMLElement {
       return;
     }
 
+    if (this._hass && computeRTL(this._hass)) {
```

Comment:
This should be done in `updated` and only if the new computedRTL value is different than the old one.

Reply:
Need help. Once I derive from LitElement it doesn't show up. Everything SEEMS to be rendering fine with no code changes but in browser the element is 8px high and doesn't show. I assume I should be also converting some internal methods to use render, etc but I need to see an example. Which class should I look at and compare to older versions? (remember this class creates a card element dynamically so it's not static).

Reply:
So I found a very close sample - `hui-unused-cards.ts` which uses LitElement and card creation internally. After I refactored hui-card-preview.ts for some reason createCardElement renders an empty card even though the configuration is given properly. The internal render method of the card is not called (and I off course didn't touch createCardElement or the entities card code).
I need help here please. How can I send you the updated code? Don't want to commit something that doesn't work.

Reply:
@balloob Refactored to LitElement and fixed based on your comment.